machida/machida.pony

```diff
@@ -636,7 +701,36 @@ primitive Machida
     r
 
   fun initial_state(computation: Pointer[U8] val): PyState =>
-    PyState(@initial_state(computation))
+    let s = @initial_state(computation)
+    print_errors()
```

Comment:
:+1: 
I wonder if we can turn this pattern into a macro 
```pony
// do some computation
let s = ...
if print_errors() then Fail() end
s
```

Reply:
It might even be worth using `FatalUserError("python error encountered")` or maybe adding a new primitive to `mort` or support for a `msg: String` parameter in `Fail()` or something.